tests/fallback_test.py

```diff
@@ -59,17 +75,28 @@ def start_client(self, port: int = None, name: str = None):
             manager=self.process_manager,
             name=name or framework.xds_flags.CLIENT_NAME.value,
             port=port or get_free_port(),
-            url="xds:///listener_0",
+            url=f"xds:///{_LISTENER}",
             image=framework.xds_k8s_flags.CLIENT_IMAGE.value,
         )
 
-    def start_control_plane(self, name: str, index: int, upstream_port: int):
+    def start_control_plane(
+        self, name: str, index: int, upstream_port: int, cluster_name=None
+    ):
         logger.debug('Starting control plane "%s"', name)
         return framework.helpers.docker.ControlPlane(
             self.process_manager,
             name=name,
             port=self.bootstrap.xds_config_server_port(index),
-            upstream=f"{_HOST_NAME.value}:{upstream_port}",
+            initial_resources=framework.helpers.xds_resources.build_listener_and_cluster(
+                listener_name=_LISTENER,
+                cluster_name=(
```

Comment:
Can change to `cluster_name=cluster_name or f"initial_cluster_for_{name}"`

Reply:
Done